chainer/functions/basic_math.py

```diff
@@ -12,14 +12,24 @@
 # Arithmetic
 # ------------------------------------------------------------------------------
 
+def _vectorize(x):
```

Comment:
We might want to use this function from other functions. Should we move it to some public place, e.g. chainer.utils?